Ecosystem Partners/Marketplace Guides/getting-started-with-ca-ppm-on-linux.markdown

```diff
@@ -0,0 +1,82 @@
+{{{
+  "title": "Getting Started With CA PPM on Linux",
+  "date": "11-13-2015",
+  "author": "Ben Hill",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+
+![](http://www.infotechgroup.com/wp-content/uploads/2014/08/CA-Technologies-logo.png)
+
+### Technology Profile 
+
+CA Project and Portfolio Management (CA PPM) helps business leaders to deliver the “right” products and applications to market faster and with less risk. Only CA PPM connects strategy to execution across the entire enterprise portfolio with relevant and immediate visibility needed to improve project ROI. - See more at [www.ca.com](http://www.ca.com).
+
+
+### Contact CA Technologies 
+
+- [Email & phone support](https://www.ca.com/us/contact/call-me.aspx)

```

Comment:
other ca.com links are http.  Please fix inconsistencies.  It applies to other files as well.